.travis.yml

```diff
@@ -22,8 +22,8 @@ branches:
   only:
   - master
   - development
-  - release/*
-  - hotfix/*
+  - release\/.*
+  - hotfix\/.*
```

Comment:
I think in travis u need to encapsulate regex with two slashes:
```suggestion
  - /^release\/.*$/
  - /^hotfix\/.*$/
```

Reply:
lets try it out